spring-cloud-alibaba-dependencies/pom.xml

```diff
@@ -18,9 +18,9 @@
     <description>Spring Cloud Alibaba Dependencies</description>
 
     <properties>
-        <revision>2022.0.0.0-RC1</revision>
+        <revision>2022.0.0.0-RC2</revision>
         <sentinel.version>1.8.6</sentinel.version>
-        <seata.version>1.6.1</seata.version>
+        <seata.version>1.7.0-native-rc1</seata.version>
```

Comment:
It seems that this version (1.7.0-native-rc1) currently only supports local installation.

Reply:
> It appears that version `1.7.0-native-rc1` currently only supports local install.

Seata community released relevant version today, the PR can be merged.